src/status_im/ui/screens/chat/views.cljs

```diff
@@ -423,6 +423,39 @@
        :inverted                     (when platform/ios? true)
        :style                        (when platform/android? {:scaleY -1})})]))
 
+(def show-floating-scroll-down-button (reagent/atom false))
+(def threshold-percentage-to-show-floating-scroll-down-button 75)
+
+(defn scroll-to-bottom []
+  (when @messages-list-ref
+    (.scrollToOffset @messages-list-ref #js {:y 0 :animated true})))
+
+(defn floating-scroll-down-button [show-input?]
```

Comment:
Hi @smohamedjavid, Is this button just a placeholder for now? Because scroll to the bottom will be part of the floating shell button component. https://github.com/status-im/status-mobile/issues/14071
![image](https://user-images.githubusercontent.com/17097240/194839281-c55fb324-f683-48ea-a098-39976eea87aa.png)

Reply:
shouldn't jump to be global and hidden when keyboard is open ?

Reply:
Yes, the floating shell button also has variants without a jump-to button, so we can remove them when opening the keyboard.

<div align="center">
<img src="https://user-images.githubusercontent.com/17097240/194850922-8ac75ea4-57df-4bd2-af62-4e6ebbf7931b.png" alt="Screenshot" height="200"/>
</div>


> global

I am not sure If I understood this correctly, but as we saw with the switcher button, it's a little hard to make it global and cover all positions and behavior of all screens.
So It will be just an abstract UI component accepting the on-press call and its visual appearance and content we have to control as per the screen, (as we do for any other component). For example, the browser tab will not have mention button.

Reply:
ok, cool, thanks, i think @smohamedjavid didn't know about jump to button , i think that's fine to merge it like this and then we could implement use it in 14071 ?

Reply:
yes, sounds good to me. Thank you

Reply:
@flexsurfer - Yes, I didn't know about the other task for floating shell buttons. I'm fine with merging this current PR as a placeholder.